sdks/python/tox.ini

```diff
@@ -287,3 +287,12 @@ deps =
 commands =
   yapf --version
   time yapf --diff --parallel --recursive apache_beam
+
+[testenv:py3-dependency-check]
+# TODO: botocore, a part of [interactive], wants docutils<0.16, but Sphinx
```

Comment:
Is there a jira for this?

Reply:
Yes, I've just created it: https://issues.apache.org/jira/browse/BEAM-10425